src/Microsoft.VisualStudio.Editors/PropPages/PackagePropPage.vb

```diff
@@ -186,35 +241,35 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
                     datalist.Add(data)
                     data = New PropertyControlData(105, "Copyright", Copyright, ControlDataFlags.None, New Control() {CopyrightLabel})
                     datalist.Add(data)
-                    data = New PropertyControlData(107, "PackageLicenseUrl", PackageLicenseUrl, ControlDataFlags.None, New Control() {PackageLicenseUrlLabel})
+                    data = New PropertyControlData(106, "PackageLicenseExpression", PackageLicenseExpression, ControlDataFlags.None, New Control() {PackageLicenseLabel})
```

Comment:
Should the control be renamed to `PackageLicenseExpressionLabel`?

Reply:
No, it was just referencing the wrong control. It should have been the expression label.